feature/experimental/bgp/ate_tests/bgp_long_lived_graceful_restart/bgp_long_lived_graceful_restart_test.go

```diff
@@ -949,12 +956,16 @@ func TestTrafficWithGracefulRestartLLGR(t *testing.T) {
 		var bgpIxPeer []*ixnet.BGP
 		t.Run("configure 5 more new BGP peers", func(t *testing.T) {
 			configureDUTNewPeers(t, dut, dutNbrs)
-			bgpIxPeer = configureATENewPeers(t, topo, ateIntfList)
+			if len(ateIntfList) > 0 {
+				bgpIxPeer = configureATENewPeers(t, topo, ateIntfList)
+			}
 		})
 
 		t.Run("Remove newly added 5 BGP peers", func(t *testing.T) {
 			removeNewPeers(t, dut, dutNbrs)
-			removeATENewPeers(t, topo, bgpIxPeer)
+			if len(ateIntfList) > 0 {
```

Comment:
From what I see, `ateIntfList` is initialized only once during the test execution and not modified after that. Why do we need multiple additional checks on this variable?